content/blog/2020-sponsor-announcement.md

```diff
@@ -0,0 +1,39 @@
+Title: Call for Sponsors is now open for PyCon India 2020
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
```

Comment:
One thing I saw that in the previous year's blog post that the author mentioned was PyCon India Content Team and so for the other PyCon conferences. So probably we should as well change this and in the previous blog post.

Reply:
@sayanchowdhury I have created that feature wherein all the posts have to have an author's name to be mentioned for proper credit back to the writer of the blog post. Whereas the default value for the field is `PyCon India Content Team` which I thought could be consistent for past year's conferences.

In the past there was no author field at all. Here's the issue #265 and the PR #263 for discussions
Here's a PyCon US blog having author's name https://pycon.blogspot.com/2020/07/recap-of-pycon-2020-converting-to-online.html

Reply:
I think it gives folks more incentive to write as they can get their attribution for the blogs they write. I mean, ofc they're written by the Content Teams, that's what the content teams are for. but I'm fine with keeping it to Content Team if you want. I can change it for the other blogs too. CC: @vipulgupta2048

Reply:
The reason, I would not have a credit back system and rather give the credit to the PyCon India <WG>Team because this brings imbalance with the other working groups. Given all the working groups are equally hard, from designers to website developers to social media team etc it's either we give attribution to all or we use name PyCon India <WG> Team. 

Giving attribution will create chaos and added logistics hence, it's better to have PyCon India <WG> Team.

Reply:
Cool, will resolve this and discuss more later. Rereading again for approval and merging